recipes/natcap-invest/meta.yaml

```
@@ -0,0 +1,75 @@
+{% set name = "natcap.invest" %}
+{% set version = "3.10.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/natcap/invest/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 0d9ce7ab5a9184c1cfcb2d2ec8295216e163b61e979b6bba3246342400945bdb
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python
+    - cython
+    - numpy
+    - babel
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - cython
+    - numpy
+    - babel
+    - pip
+  run:
+    - python
+    - cython
+    - gdal >=3.1.2,!=3.3.0  
+    - pyro4 ==4.77
+    - pandas >=1.2.1
+    - numpy >=1.11.0,!=1.16.0,<1.22.0
```

Since this builds against the C interface of numpy (inclusion in `host`), I think you need to also add a `pin_compatible` statement here in addition to the runtime constraints that upstream wants to put on numpy.
```suggestion
    - numpy >=1.11.0,!=1.16.0,<1.22.0
    - {{ pin_compatible('numpy') }}
```
Then again, the build output also says
```
WARNING (natcap.invest): plugin library (Python) package conda-forge::numpy-1.18.5-py38h8854b6b_0 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
```
so I'm not sure if numpy is actually making it into the python extension objects and then this might not be necessary.